pkg/policies/opa/rego/docker/docker_run/runusingupgrade.rego

```diff
@@ -0,0 +1,8 @@
+package accurics
+
+{{.prefix}}{{.name}}{{.suffix}}[cmd.id]{
+	cmd := input.run[_]
+    config := cmd.config
+    commands = ["apt-get upgrade", "apt-get dist-upgrade"]
+    contains(config, commands[_])
```

Comment:
value of `config` variable may or may not be in form of an array, need to create checks for both possibilities. please keep edge cases in mind while writing policy code